package.json

```diff
@@ -26,7 +26,8 @@
   "scripts": {
     "prerun": "npm run-script build",
     "run": "jpm run --addon-dir dist/ ${npm_config_jpm_runflags} --prefs jpm-prefs.json",
-    "test": "mocha-webpack --opts ./mocha-webpack.opts --webpack-config ./webpack.config.test.js -r test/setup.js test",
+    "test": "cross-env NODE_ENV=test mocha-webpack --opts ./mocha-webpack.opts --webpack-config ./webpack.config.test.js -r test/setup.js test",
+    "coverage": "nyc --all --include \"src/**/*.js\" npm test",
```

Comment:
I really don't agree with this breakout.

I sort of understand the concern about performance,but only sort of.

Reply:
I don't have a strong opinion here, but I split them out just so that it's easier to see the test results during dev.

Reply:
I re-merged `coverage` and `test` since it'll probably change anyway when we switch to using karma.

Reply:
👍